lib/src/model.dart

```diff
@@ -1344,26 +1343,47 @@ class Extension extends Container
         ElementType.from(_extension.extendedType, library, packageGraph);
   }
 
-  /// Returns [true] if there is an instantiation of [c] to which this extension
-  /// could be applied.
-  bool couldApplyTo(Class c) =>
-      _couldApplyTo(extendedType.type, c.element, packageGraph.typeSystem);
+  bool couldApplyTo(Class c) => _couldApplyTo(c.modelType);
 
-  static bool _couldApplyTo(
-      DartType extendedType, ClassElement element, Dart2TypeSystem typeSystem) {
-    InterfaceTypeImpl classInstantiated =
-        typeSystem.instantiateToBounds(element.thisType);
-    classInstantiated = element.instantiate(
-        typeArguments: classInstantiated.typeArguments.map((a) {
-          if (a.isDynamic) {
-            return typeSystem.typeProvider.neverType;
-          }
-          return a;
-        }).toList(),
-        nullabilitySuffix: classInstantiated.nullabilitySuffix);
+  /// Return true if this extension could apply to [t].
+  bool _couldApplyTo(DefinedElementType t) {
+    if (t.element.name == 'Implementor2' && name == 'ExtensionCheckLeft') {
+      print('hi');
```

Comment:
Dubugging code that should be removed?

Reply:
Eventually I will figure out how to make conditional breakpoints more persistent than they seem to be in intellij, but not so persistent that I forget to remove them when uploading PRs.  \*sigh\* Fixed.